docs/model.md

```diff
@@ -331,6 +331,38 @@ query {
 }
 ```
 
+### Defining custom mutations
```

Comment:
can you keep documentation consistent by adding method name as a header?

Reply:
constitency is saved!